core-primitives/stf-state-handler/Cargo.toml

```diff
@@ -51,10 +51,10 @@ rust-base58 = { package = "rust-base58", version = "0.0.4", optional = true }
 thiserror = { version = "1.0", optional = true }
 
 # scs / integritee
-sgx-externalities = { default-features = false, git = "https://github.com/integritee-network/sgx-runtime", branch = "master", optional = true }
+sgx-externalities = { default-features = false, git = "https://github.com/ajuna-network/sgx-runtime", branch = "update-substrate", optional = true }
```

Comment:
The update-substrate branch was not merged to master on purpose?

Reply:
i created a pr in sxg-runtimte: https://github.com/ajuna-network/sgx-runtime/pull/2